app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = @user = User.includes(posts: :comments).find(params[:user_id])
     @posts = @user.posts.includes(:comments)
```

Comment:
- [ ] Dear @hmunish, It is crucial that you add In your PR description the screenshots of your app's logs before and after the fix while dealing with N+1. [Refer here for requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Do%20not%20use,after%20the%20fix).